src/Core/src/Handlers/RefreshView/RefreshViewHandler.Android.cs

```diff
@@ -66,5 +66,26 @@ public static void MapContent(IRefreshViewHandler handler, IRefreshView refreshV
 
 		public static void MapRefreshColor(IRefreshViewHandler handler, IRefreshView refreshView)
 			=> UpdateRefreshColor(handler);
+
+		public static void MapIsRefreshEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
+			=> UpdateIsRefreshEnabled(handler);
+
+		public static void MapIsEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
+			=> UpdateIsEnabled(handler);
+
+		static void UpdateIsRefreshEnabled(IRefreshViewHandler handler)
+		{
+			handler.PlatformView.Enabled = handler.VirtualView.IsRefreshEnabled;
+		}
+
+		static void UpdateIsEnabled(IRefreshViewHandler handler)
+		{
+			// On Android, IsEnabled should disable the entire view tree
+			handler.PlatformView.Enabled = handler.VirtualView.IsEnabled;
+			
+			// Also update the refresh enabled state
+			if (handler.VirtualView.IsEnabled)
+				UpdateIsRefreshEnabled(handler);
```

Comment:
Not sure we need this as disabling the control disabled EVERYTHING.  In face, we don't need to have this method at all since the base ViewHandler has it already nd we are not adding anything new to it.